parts/ch11.tex

```diff
@@ -117,35 +124,38 @@ \subsection{Определенность в обработке зрительн
+\subsection{Активные символы}
 
-Во-первых, важно подчеркнуть, что символы бывают либо дремлющие, либо активированные. Активированный символ --- это тот, нейроны которого возбудились, когда внешние стимулы превысили определенный порог. Поскольку символ может быть возбужден различными способами, став активным, он может действовать по-разному. Это значит, что мы должны думать о символе не как о застывшей, но как о меняющейся, динамической единице. Таким образом, описывая состояние мозга, недостаточно сказать «символы А, В,\ldots{} N в данный момент активны»; вместо этого необходимо представить набор параметров для каждого активного символа, характеризуя некоторые аспекты того, как этот символ работает «изнутри». Интересен следующий вопрос: есть ли у каждого символа некие центральные нейроны, посылающие сигналы каждый раз, когда символ активизируется? Если такая «сердцевина» существует, мы могли бы называть её «неизменной сердцевиной» символа. Соблазнительно предположить, что каждый раз, когда вы думаете, скажем, о водопаде, повторяется некий один и тот же нейронный процесс, хотя и видоизмененный немного в зависимости от контекста. Однако неясно, так ли это происходит на самом деле.
+В дальнейшем давайте называть эти гипотетические нейронные комплексы, нейронные модули, нейронные группы, нейронные сети и мультинейронные единицы \emph{символами}, как бы они не выглядели --- как блинчики, грабли, гремучие змеи, снежинки или даже как волны на воде. Описание состояния мозга в терминах символов уже было упомянуто в Диалоге. На что походило бы подобное описание? Какие типы понятий можно представить себе в виде символов? Каким образом подобные символы были бы связаны между собой? И как вся эта картина поможет нам понять, что такое сознание?
+
+Во-первых, важно подчеркнуть, что символы бывают либо дремлющие, либо активированные. Активированный символ --- это тот, нейроны которого возбудились, когда внешние стимулы превысили определенный порог. Поскольку символ может быть возбужден различными способами, став активным, он может действовать по-разному. Это значит, что мы должны думать о символе не как о застывшей, но как о меняющейся, динамической единице. Таким образом, описывая состояние мозга, недостаточно сказать «символы А, В, \ldots{} N в данный момент активны»; вместо этого необходимо представить набор параметров для каждого активного символа, характеризуя некоторые аспекты того, как этот символ работает «изнутри». Интересен следующий вопрос: есть ли у каждого символа некие центральные нейроны, посылающие сигналы каждый раз, когда символ активизируется? Если такая «сердцевина» существует, мы могли бы называть её «неизменной сердцевиной» символа. Соблазнительно предположить, что каждый раз, когда вы думаете, скажем, о водопаде, повторяется некий один и тот же нейронный процесс, хотя и видоизмененный немного в зависимости от контекста. Однако неясно, так ли это происходит на самом деле.
```

Comment:
I would typeset "A,B,...,N" as "`A,~B,\dots,~N`". Here, \dots would auto-convert to \dotsc (however, I'm not sure about it, since it is text and not math...). \ldots is also OK.